includes/payment-methods/class-wc-stripe-express-checkout-ajax-handler.php

```diff
@@ -135,9 +141,9 @@ public function ajax_clear_cart() {
 		WC()->cart->empty_cart();
 
 		if ( $booking_id ) {
-			// When a bookable product is added to the cart, a 'booking' is create with status 'in-cart'.
+			// When a bookable product is added to the cart, a 'booking' is created with status 'in-cart'.
 			// This status is used to prevent the booking from being booked by another customer
-			// and should be removed when the cart is emptied for PRB purposes.
+			// and should be removed when the cart is emptied for ECE purposes.
```

Comment:
Non-blocking nit: I think we should use the full term rather than `ECE`.
```suggestion
			// and should be removed when the cart is emptied for express checkout purposes.
```